src/etc/inc/voucher.inc

```diff
@@ -31,20 +31,29 @@ if (!function_exists('captiveportal_syslog')) {
 	require_once("captiveportal.inc");
 }
 
-function xmlrpc_sync_voucher_details(&$syncip, &$port, &$username, &$password) {
-	global $config;
-
-	if (!empty($config['voucher'][$cpzone]['vouchersyncdbip'])) {
-		$syncip = $config['voucher'][$cpzone]['vouchersyncdbip'];
-		$port = $config['voucher'][$cpzone]['vouchersyncport'];
-		$username = $config['voucher'][$cpzone]['vouchersyncusername'];
-		$password = $config['voucher'][$cpzone]['vouchersyncpass'];
+function xmlrpc_sync_get_details(&$syncip, &$port, &$username, &$password, $carp_loop = false) {
```

Comment:
This function name is a bit too generic for where it is / what it does. It's specific to captive portal, so it should have some indication of that in its name.